ros/src/aero_offboard_velocity/src/aero_offboard_velocity_node.cpp

```diff
@@ -1,125 +1,196 @@
 /**
  * @file mavros_offboard_velocity_node.cpp
- * @brief MAVROS Offboard Control example node, written with mavros version 0.14.2, px4 flight
+ * @brief Aero Offboard Velocity example node, written with mavros version 0.22.0, px4 flight
  * stack and tested in Gazebo SITL & jMAVSIM.
  */
 
 #include <cstdlib>
-#include "aero_drone_action.h"
-bool offboardCtrlNed(AeroDrone aero)
+#include "aero_drone_offboard.h"
+
+void stepUp(AeroDrone &aero);
+void stepDown(AeroDrone &aero);
+bool offboardVelocityCtrlNED(AeroDrone &aero);
```

Comment:
Can you please add `offboardVelocityCtrlNED()` and `offboardVelocityCtrlBody()` first ? As `stepUp()` and `stepDown()` are utility methods for the former ones.
```
bool offboardVelocityCtrlNED(AeroDrone &aero);
bool offboardVelocityCtrlBody(AeroDrone &aero);

// Utility methods
void stepUp(AeroDrone &aero);
void stepDown(AeroDrone &aero);
```